src/ray/raylet/node_manager.cc

```diff
@@ -1354,8 +1358,9 @@ void NodeManager::HandleWorkerAvailable(const std::shared_ptr<WorkerInterface> &
   }
 }
 
-void NodeManager::ProcessDisconnectClientMessage(
-    const std::shared_ptr<ClientConnection> &client, bool intentional_disconnect) {
+void NodeManager::DisconnectClient(const std::shared_ptr<ClientConnection> &client,
+                                   rpc::ClientDisconnectType disconnect_type) {
+  RAY_LOG(ERROR) << "DisconnectClient*****" << disconnect_type;
```

Comment:
Remove debug log.

Reply:
this commit was not pushed to the branch 🤦  fixed